thirdparty/jsoncpp/devtools/batchbuild.py

```diff
@@ -0,0 +1,278 @@
+from __future__ import print_function
```

Comment:
the tests source code are there because it was from the drop for jsoncpp. but i disable build and run the same in the minifi build for the jasoncpp cmake.